src/EditorFeatures/TestUtilities/Workspaces/DeferredDocumentationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
```

Comment:
this is effectively a move.   this was the type we used in the workspace to supply doc comments over one compilatino to another language.  however, it forced the compilation to both be built and to stay alive.  We want to moe away from that and just persist to disk.  However, this type is still useful for testing purposes.